src/Sylius/Bundle/UiBundle/Twig/Component/TemplateLivePropTrait.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Sylius\Bundle\UiBundle\Twig\Component;
+
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+
+trait TemplateLivePropTrait
```

Comment:
```suggestion
trait TemplatePropTrait
```
The `Live` part might suggest that it can only be used with live components.